docker-compose.yml

```diff
@@ -2,15 +2,14 @@ version: '3'
 services:
   decidim:
     image: decidim/decidim:latest-dev
-    entrypoint: []
+    entrypoint: /usr/local/bin/entrypoint.sh
     command: decidim
     working_dir: "/code"
     ports:
       - "3000:3000"
     volumes:
       - .:/code
-      - bundle:/usr/local/bundle
```

Comment:
Was mounting `/usr/local/bundle` giving some issues? It dramatically improves performance on mac (as we're not synching with a real filesystem, which is the cause of the slowdown)

Reply:
Yeah, the change seems only to catch up with the changes in the base image (which now install to `/gems` instead of `/usr/local/bundle`), it doesn't do anything differently.

I'm surprised though that having the `gems` folder syncronized would cause performance issues. I've been using similar changes to develop on docker, but installing gems to `./.bundle` instead (inside the repo folder) and removing the mount (similarly to `npm`'s behavior installing to `node_modules`). It worked fine without any noticiable slowdowns. That folder would only change on `bundle install`, `bundle clean` or things like that, so the syncronization workload seems very small... This approach has the upside of allowing the developer to easily debug code inside gems, for example.

Reply:
@deivid-rodriguez I did similar things as well and never experienced performance issues in the past.

Reply:
@josepjaume I've been investigating a bit about your issue.

I've made a small change that is supposed to improve performance for mac users. The idea was taken from

https://github.com/docker/for-mac/issues/1592
and
https://github.com/docker/for-mac/issues/77